cli/internal/cmd/create.go

```diff
@@ -83,6 +85,9 @@ func (c *createCmd) create(cmd *cobra.Command, creator cloudCreator, fileHandler
 	if err != nil {
 		return err
 	}
+	if err := validateCLIandConstellationVersionAreEqual(constants.VersionInfo(), conf.Image, conf.MicroserviceVersion); err != nil {
+		return err
+	}
```

Comment:
I feel like it would be nice to be able to skip this validation (or make it a warning) when `--force` is set (`flags.force`).